facia-purger/src/main/scala/com/gu/purge/facia/Lambda.scala

```diff
@@ -54,29 +46,15 @@ class Lambda extends RequestHandler[S3Event, String] {
       .header("Fastly-Soft-Purge", "1")
       .post(EmptyJsonBody)
       .build()
-    if (STAGE == "PROD") {
+
+    if (stage == "PROD") {
       val response = httpClient.newCall(request).execute()
       println(s"Sent purge request for content with ID [$contentId]. Response from Fastly API: [${response.code}] [${response.body.string}]")
```

Comment:
basically all stdout/stderr goes to the lambda log, if you need more control you can send to kinesis/s3/wherever.  I feel like we shouldn't go overboard as lambdas are supposed to be simple, but it would be nice to have it all in the same place.